src/test/scala/org/kibanaLoadTest/helpers/Response.scala

```diff
@@ -0,0 +1,56 @@
+package org.kibanaLoadTest.helpers
+
+case class Response(
+    userId: String,
+    name: String,
+    status: String,
+    method: String,
+    url: String,
+    requestHeaders: String,
+    requestBody: String,
+    responseStatus: String,
+    responseHeaders: String,
+    responseBody: String,
+    requestSendStartTime: Long,
+    responseReceiveEndTime: Long,
+    message: String,
+    requestTime: Long
+) {
+  override def toString: String =
+    s"$userId - $name - $status - $method - $url $requestHeaders - $requestBody - $responseStatus - $responseHeaders" +
+      s" - $responseBody - $requestSendStartTime - $responseReceiveEndTime - $responseReceiveEndTime - $message - $requestTime"
```

Comment:
Just a comment, feel free to ignore, but I think this could be:
```
override def toString: String = 
"""$userId - $name - $status - 
|$method - $url - $requestHeaders - $requestBody - $responseStatus - 
|$responseHeaders - $responseBody - $requestSendStartTime - 
|$responseReceiveEndTime - $responseReceiveEndTime - $message - 
|$requestTime""".stripMargin
```

I found the syntax here: https://www.oreilly.com/library/view/scala-cookbook/9781449340292/ch01s03.html

This code doesnt look bad at all, I just like playing with syntax sometimes.

Reply:
Agree. I actually use the same style in other place. Changing